### ISSUE TITLE
Make sure python modules use _ instead of - and may be Json files should follow the same

### ISSUE THREAD

**Issue by henrykironde** [weecology/retriever#1119]
Based on the namespace standards, Package and Module Names should have short, all-lowercase names. Underscores can be used in the module name if it improves readability not in the package name. 

I think all Json files should use the _ instead of - for consistency.
We have a file that tests Json scripts, we can add some procedures to keep that uniform if we agree to that.

**Comment by zhangcandrew** [comment]
Do you mean like changing the output from retriever ls for json files? I think the names of the json files already use _, but I think I might be missing something here.

**Comment by henrykironde** [comment]
We have this right now. I also thought all use _ but we have some
```python
ls ../retriever/scripts/*-*
../retriever/scripts/breed-bird-survey-nlcd.json	../retriever/scripts/ngreatplains-flowering-dates.json
../retriever/scripts/globi-interaction.json		../retriever/scripts/partners-in-flight.json
../retriever/scripts/nd-gain.json			../retriever/scripts/portal-dev.json
```

**Comment by zhangcandrew** [comment]
Oh I see, I didn't realize that! Ok, I think it makes sense to keep naming consistent with _ instead of -. The file that tests JSON scripts you were talking about, is that test_json.py?

**Comment by henrykironde** [comment]
Yes for json file test_json.py  and make sure filename does not have -
And another test on python filenames, this could be a new file or we can add it to test_json.py

**Comment by ethanwhite** [comment]
Fixed in 03d20e65a19d00579066824d94f5b72eea477f9d.
